src/common/components/signUpActivationModal/signUpActivationModal.component.js

```diff
@@ -0,0 +1,131 @@
+import angular from 'angular'
+import includes from 'lodash/includes'
+import sessionService, { Roles, createAccountDataCookieName } from 'common/services/session/session.service'
+import orderService from 'common/services/api/order.service'
+import template from './signUpActivationModal.tpl.html'
+import cartService from 'common/services/api/cart.service'
+
+const componentName = 'signUpActivationModal'
+
+class signUpActivationModalController {
+  /* @ngInject */
+  constructor ($log, $scope, $cookies, $window, sessionService, orderService, cartService, envService) {
+    this.$log = $log
+    this.$scope = $scope
+    this.$cookies = $cookies
+    this.$window = $window
+    this.sessionService = sessionService
+    this.orderService = orderService
+    this.cartService = cartService
+    this.imgDomain = envService.read('imgDomain')
+    this.publicCru = envService.read('publicCru')
+  }
+
+  $onInit () {
+    if (includes([Roles.identified, Roles.registered], this.sessionService.getRole())) {
+      this.onStateChange({ state: 'sign-in' })
+    }
+    if (!this.isInsideAnotherModal) {
+      this.cartCount = 0
+      this.getTotalQuantitySubscription = this.cartService.getTotalQuantity().subscribe(count => {
+        this.cartCount = count
+      }, () => {
+        this.cartCount = 0
+      })
+    }
+    this.loadingAccountErrorCount = 0
+    this.initialLoading = true
+    this.getUnverifiedAccount(false)
+    this.getUnverifiedAccountTimoutId = this.$window.setInterval(() => {
```

Comment:
I think we should be using `$interval` here and `$timeout` below

Reply:
I'll do this

Reply:
signUpActivationModal.component.js is one of the components that need to be deleted since we aren't handling that anymore.

I'll go through the code and remove any components or code we no longer need.

Reply:
This is now complete